samples/containers/poetry/src/main/java/com/squareup/sample/poetry/PoemWorkflow.kt

```diff
@@ -1,114 +1,10 @@
 package com.squareup.sample.poetry
 
 import com.squareup.sample.container.overviewdetail.OverviewDetailScreen
-import com.squareup.sample.poetry.PoemWorkflow.Action.ClearSelection
-import com.squareup.sample.poetry.PoemWorkflow.Action.HandleStanzaListOutput
-import com.squareup.sample.poetry.PoemWorkflow.Action.SelectNext
-import com.squareup.sample.poetry.PoemWorkflow.Action.SelectPrevious
 import com.squareup.sample.poetry.PoemWorkflow.ClosePoem
-import com.squareup.sample.poetry.StanzaWorkflow.Output.CloseStanzas
-import com.squareup.sample.poetry.StanzaWorkflow.Output.ShowNextStanza
-import com.squareup.sample.poetry.StanzaWorkflow.Output.ShowPreviousStanza
-import com.squareup.sample.poetry.StanzaWorkflow.Props
 import com.squareup.sample.poetry.model.Poem
-import com.squareup.workflow1.Snapshot
-import com.squareup.workflow1.StatefulWorkflow
-import com.squareup.workflow1.WorkflowAction
-import com.squareup.workflow1.WorkflowAction.Companion.noAction
-import com.squareup.workflow1.parse
-import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
-import com.squareup.workflow1.ui.backstack.BackStackScreen
-import com.squareup.workflow1.ui.backstack.toBackStackScreen
+import com.squareup.workflow1.Workflow
 
-typealias SelectedStanza = Int
-
-/**
- * Renders a [Poem] as a [OverviewDetailScreen], whose overview is a [StanzaListRendering]
- * for the poem, and whose detail traverses through [StanzaRendering]s.
- */
-object PoemWorkflow : StatefulWorkflow<Poem, SelectedStanza, ClosePoem, OverviewDetailScreen>() {
+interface PoemWorkflow : Workflow<Poem, ClosePoem, OverviewDetailScreen> {
```

Comment:
Please keep the kdoc. And maybe note that we use the interface to allow other implementations.